ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_svi_settings.schema.yml

```diff
@@ -80,6 +80,20 @@ $defs:
         items:
           type: str
           description: IPv6_address.
+      ipv4_acl_in:
+        description: |-
+          Name of the IPv4 access-list to be assigned in the ingress direction.
+          The access-list must be defined under `ipv4_acls`.
```

Comment:
We should probably update the descriptions here and on ipv4_acls (and under l3_interfaces on the node config) to clearly state which fields support replacing. In this case it is only `interface_ip`.